src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -113,8 +111,8 @@ trait Backend {
    */
   def bindCall(workflowDescriptor: WorkflowDescriptor,
                key: CallKey,
-               locallyQualifiedInputs: CallInputs,
-               abortRegistrationFunction: AbortRegistrationFunction): BackendCall
+               locallyQualifiedInputs: CallInputs = Map.empty[String, WdlValue],
+               abortRegistrationFunction: AbortRegistrationFunction = AbortRegistrationFunction(_ => ())): BackendCall
```

Comment:
So `abortRegistrationFunction: Option[AbortRegistrationFunction]`.  Can we still have a default value of `None`?